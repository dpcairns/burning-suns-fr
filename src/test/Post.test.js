import React from 'react';
import { shallow } from 'enzyme';
import Post from '../Post';

const note = {tite: 'my name', date: 'adate'}
describe('First React component test with Enzyme', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Post post={note}/>);
      expect(wrapper).toMatchSnapshot()
     });
 });