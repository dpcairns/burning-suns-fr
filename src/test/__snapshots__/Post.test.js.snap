// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<WithStyles(ForwardRef(Grid))
  item={true}
  md={6}
  xs={12}
>
  <WithStyles(ForwardRef(CardActionArea))
    component="a"
    href="#"
  >
    <WithStyles(ForwardRef(Card))
      className="makeStyles-card-1"
    >
      <div
        className="makeStyles-cardDetails-2"
      >
        <WithStyles(ForwardRef(CardContent))>
          <WithStyles(ForwardRef(Typography))
            component="h2"
            variant="h5"
          />
          <WithStyles(ForwardRef(Typography))
            color="textSecondary"
            variant="subtitle1"
          >
            adate
          </WithStyles(ForwardRef(Typography))>
          <WithStyles(ForwardRef(Typography))
            paragraph={true}
            variant="subtitle1"
          />
          <WithStyles(ForwardRef(Typography))
            paragraph={true}
            variant="subtitle1"
          />
          <WithStyles(ForwardRef(Link))>
            <WithStyles(ForwardRef(Typography))
              color="primary"
              variant="subtitle1"
            >
              Show if there is an attached city
            </WithStyles(ForwardRef(Typography))>
          </WithStyles(ForwardRef(Link))>
        </WithStyles(ForwardRef(CardContent))>
        <DeleteSharpIcon />
      </div>
      <Hidden
        xsDown={true}
      >
        <WithStyles(ForwardRef(CardMedia))
          className="makeStyles-cardMedia-3"
          image="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMTEhUTExMVFhUWGBoYGBgYGBoXGBgXHRoXFxgYGBcYHSggGhomHhcdIjEhJSkrLi4uGCAzODMtNygtLisBCgoKDg0OGhAQGi0lHyUtLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIALcBEwMBIgACEQEDEQH/xAAbAAACAwEBAQAAAAAAAAAAAAACAwABBAYFB//EADkQAAEDAwIEAwYGAgEEAwAAAAEAAhEDITESQQQFUWEicYEGEzKRofBCscHR4fEUUiNicoKSFUNT/8QAGAEBAQEBAQAAAAAAAAAAAAAAAAECAwT/xAAdEQEBAQEAAwADAAAAAAAAAAAAEQECEiExAyJB/9oADAMBAAIRAxEAPwD5gEYarDUYC9TmprUbQr07o2tWoiAXt6KwETWo2tViJSgQcwbtIsR5g7qAevn95RhiINVFOpwdiPnmCqDUxrE2k1sGZm2m1je4duLXnt3kWBICInFhYR/fdaaPC6muOpg07F0F2fhnPlO6W0kTG4gohRb0V6UwNReikCg3sj1WjcGx6dfObfLujcASTAaOmw7XQ6FQssP6+nVXUpFpggg7g27pjW9lWg/fZAsNQlqe1o3x2zOyHSoFxIF8WAnGTbsqDN+hTQ1SYEAkAgahsf4RazhhNvXCGFqbT3GwnuPL+EDGnpP7ZSAHlpaBB1Am82IOLHBBnHXtcHk6A2QRJMbg2Bv0MD5JjmdrIC1RSg0i+4O/7HKWWrQ6SqaBuJzvHl8iswZnMQOaJstBb2QEfNTWmdwQELcDOok3Ive7iSOoMn5LM4KbgQWoSnEIYF56WgxfabXCikEKIyFakwrQ0eiPSoxM0rrGFuowBcXEiD6QehsrY1EG2F0TWrQgRtATKgaYIbFgCJnxAXPYHMXyoGE2AVZqMpWJkWi03M7jr+d/NQNTABAtfrOeh7Ivdb/31x0SFM4PQCQ6ANDhOnUJI8Ntr/iFxbN5SGJjQU5jwGuaWtJMQ64LSDJiLXEiD+iqM5aMgQNv7R6bDH6+oTDRI732IKtlOTHXGB81YE6E6nRJnSDZsny/EfL72Vxt9+ittNAkNRNbbdMDUbmRn7CQJY0b/fmEJp2n+1pqTa8iI8t4+aDSgT7sRM3nEbbGcKiwStIo2JkWi25G5Fot+qFrVIM0KtC0CnP5odCkAe4JaX2gEA3AN5Pw5i2cJRCeaalTxGYA6xYT5bIrMG9fu1kJnJv5pyohRWctUqNE2mO/6wnFnzQFqilsaA7xCRNwDEjsUlzTlacb5SyPksqVUpABpDgSRcQQWmYgyIMi9uqVVYAbGbDaIO4+8pzgghQZ3NQOan6ev0/dVVAk6ZiTAMExtJGT6BSKzlRHCihWgBGwbImDsjDbwu2MJ5owB99FGtTadiN+xweyqJSpkxGf4RNancQ/W4u2m05AwASBcwM7oQ1VFaYPX8kfzVhibSgZAO0Y2sbdMqppQCYMYvMz+kY/pWAm025kC/VWCqguYjMyBH028kLWrQKNi4HETJAN+g3E9OoTazdNhp/CbQ4TGxyJ3H7IMgatT6dP3bSHHWSdTS2wA+Etdv5dktNqvc4NBg6RAgDGbkZN90GYM+8KzTT2sE3JjrG3krGTYfJIEtADSC2SYgyQRGbYIPzsPVhc0hrdGkidTp+IbSDYEde6haoW9PsoEEKNH2U0s8irqUi0wQQQi4zls3jPT9FTx2gLRMSIz1A6zY7eir3Z0ztMZ9cZ9VBlc1VokG4t9fJPiPUIVNGYt7K6kEzEJzmlC5ikVnDMzP8APRAW2WhzLIajRPhmLZufopqgYxo1B4MlvhLSLOtEjcZBGR6Qs5YFodcz9+fmlELK0l7TG90ssWnSIMmCMCM9b7JLmpqk+6kE2t3E9LDf0SSFphKeFkJ0KkxzYMEQVakGgJjRCpoTGhdWNQNRtaoGpjQtYgwG6REzfUDjsQfvHyJgOylMDf080xoVwU0I2NRmnEXB3t95TWUiYwATFzF+/wA8qoW65JgCdhj0HRExvaeysNTgIi9+yoF2mbC1s7dRZW6Zmb9d0QYia1ADmff6KaU+kz5b3iR2ULMYRIWxgJAJgdcwOsKEfYx6TsmhkmDb7yjDLHFj1/IbhRSRIEbEzsbj+/X0QFq0CINrmIPTqghAt99o7D+UOiVocBsqe0TYyNrR9EGcYjzUeLxY7D739U0hC4KKSGi0/fkgqMgkdPu60vAgdd+/Qqiw6ZiQbaj18/vCDJpTaPDhwJLmNgXkmSbkQBftbCsCCCD/AAUtwUCHMug0/stBbY4PfsgqxaAcDJm+/os6rK5WylIcZb4RgmCQTFhvHb8pjRTqluqDGoFptMg/f0SozYYMZ+kbqKyFqAtT3oHtUUktsbevRA6o7Tpk6Z1RtOJjrCfVZEiQTO2PQpD2qaJV4h7jLnucYAkuJMAAASegEeiiCFEuq0NTGtVsaI31TjaI69Z/NG1ll0xhbQmNpmJ/v5dFKdj+yYAR1VxFNHVNa2cSrYG6TY6pEHaLyCOuPqi7qg6DLgwDfBwe1vu61VKDNQ0vEEasHwG/gM5IiJ3t6ZvSE9jb7Hufz6qoEMNrWMx3RNE26nP9onU4MSDByMHuCrQHoAcQ7aRIjIx6IAEQCINQE4yBaB656+eJVAI6gbPhmLZz3U0oBLUwUfDq7xkfKPvdW1vRTSgpzR5yPlfB6/0hc377JgCilC3tE9JQwTe56+ScWlLjpM/YQLLd0EJ2lDCmtElqo4hNOEOlEIj76IHScx+S18QwWhpaCLSZn13vb0WctUCXC2AgqMI2jf8AlOePJCANzbtkdMqKQQCCZAI2vfr6pMLTqibAz2+4Si1TRncFKpJuSSbC/QWH7Junt97oH7dvv5qRWZwQOxEDz38lonNgZ+5HdAIgiBeLmZHlH8qKM1qX/wCRwPxA3i94G/2VFm0qK+QfeU1oQgJjQtsjpsnp62+ytIpu0gkjS4mCYMEZmLjI+iQAExoWkPFMRh2LG0TIB9P4VMCgNkxrDE7fn1j5hEHRoudJa1x0iTAmANz0CJuRH5forpvIuCRtYxbB/NEWFpgggjIIgj0KCACN5/nZHT8pVU3ROLiLic/r3VsQQNRtYoAraUqrARAKw6MX8wqBSoIBXpWelxALiOn5g3/MLS1/c3+oypVUBZG+lG4MgGxxP5EdCqVvAm1xt/SULIQgwexsYtI6I5MRtn7+aFzcFKijTMnSCQJPW3Ux+apgBsbCRJiSNjH7K9UGx+8FUfvupQsj76oCEx7fXyQFRQPugewWg7XERB/UJhQkJQgtQPH3/Kc9v30QPZFjaVFJf0x1SnBPe3ughRSw1zdLxImYIO4zjBv9e6SBYiPp+XRNQiLz0+v6IMzghq0SACQQHCxIsbwYO901wS39JssKRCiMhRCHAJzAqYOqYAurIzczAHlhOiLIGJjQqgmN9eq0AkgibC8E+kgbmEo5sI7LVX4TQGHU0626oGRciCOtkQkJtSo5xlxLjYSTJgCAJPYIAFYSiwEUfyo1QpRYTKrwTZunFpJvFzfqlgqB8GUUx1vW/wDaRX4jS03IBEEiLDrdbaHBvqk+7a50DUYGOuNlz3PK8NeyoPAWOh+BP+pjFzk9lN2GPM5RzMninUy6dIuf9rBs3iLAExP6rr2O8l8k5fxTqRDy8CSBiTDcifwiwHyXd+z/ADQVAA3AJmbF0knVpknpvuei4/j7/jfXLpQf2VakDXi8/TqoMLrXNb6lxOXE/OCf0U1EGQcLw+c8cW1aTGySXCRgAf7T6x0Ppf16VUEkTiJi8E3i+bR81PL3Fgiqm33ZQlVUi0Ti89d47ffdKKMd8fX9lg5hXg07E+IiAL3g3Pk02K1lcvzHnYbWa2ILCS/NxcCI/Wyz10uZXSyovHZzoOcwBrjrEgwYg7kjaL+oXr7K51fhAkoXBHCrVE4++iUJIQu6WTCEBCVSqjRaPXz/AFS3J7yTAJsMdt0lzVkJcgFvqnPbFkpwU1SSojfTIJBEEZByop7Ke1q0UqYgk9YA3n9kgJtMLtWdw2mm02E2AJ8rmBcoA75G8I2OIgixG4yrQelMDTlZuIeWsLhtf907h32BnYGf1UqGsZJUCrooEpBNKk2N/L9lTj1Wfi6xZBA1N/FGQL+LuOybqwvgOKLi8REOvve2CNrLY5cvwftE9zhSbSbqL3ybgATEE9oLsTePLpgVjnqruHUeLqMB0VHMLrS0wd1w/Ee0LRV/5mEPpwA4AanWcHSDYNNjBG82Xs+0IqMb72mYIzJIvgRsN/NcHzSuKji4karSQI1uMku7Dssfk6a55BwlBlSvpe9rGucfEAdOT8IIsPOIC6TgOcUqbmU6DfA0wca6pE+MFt9iLjDjiFxq6/2O5cHt965wAYSTYnwgfDOGi5Nrndcud2ta73h3G0nzMdc2Q8ZWDGuMghu+Afnie68yjz6lr0nBswz8ZmPDHT5J3+a3UaNbSCbAHDpjw3EE32Xfzco5/mHMWVanxR7tr9RjSBqLQ3UCbwCRF/qvX9n+J1TEgEagI2sJcf8AY5+S8D2oY2i9ssp3a8NHiIjU3SC0biTHy7r1PZ8U2fDokCXu8WrFgdV53juud/Zvfjo3FC4rNxHED3evHibANj8QG4Wgn5kT2+a6ViMfM6jxTJYJIvAsSJuB3hcP7U1Q5we0yMTEh0/C0kWlt7dxbc9/xIBxMRgx6+YXyrnJHvHtYIbq+ESADvAJ8lz71vh0HslwzGONQva7SyO7TLiQPQTHddfwzyWgkRO3RfNOSUX1qgphxALpMZm/igfKdpX0Xl9Wm5ummdTWeGRcSNp3U41emsKOb9VAUvinQ1x6NK6VgWkeSAhExwdj7KhSqS4IHtTiqlukgtJJiDMRmQRvPpEKUhFRt4F/TPokvblNclPCm6pRCitRT0sPZfoExiBhytAqDTGkTM6t4iIIwdj812rMWAiahaUwFKRm43imQ5hIBg5npm2AMz3QcveGUmTE6QB4gScNF/UeUry/aWhA940mWglzQblsEOInEAk27K6NI1XguqtDWQWtaQ5zf+4x0HzJXPd2rHSypKBjYAgWx28kUre6kU51l4XF89HuHVDScWhxY5piRkTuItHRexxtdtOmXnVLQSRGekH6L53zHhXNLnNaTTqiWObMaSHOg7yIv5G11jvqfFzDeTc/NJ7pJcL6dRN9TpJdJ9fRdnwzX1ml5c5gePAG20ibOncn5QvmFM6bm9pEEeQJzvsvqfs+D/j05yWgmReTf0yscavWMXF8U2pTqUaj/dVGj4niGPxfBlp3HfK+f8yoOY+XAAPAe3EFrrggbA9DddF7Qc0oVQ8PbFZh06ocQ4CRgEQ4Xzid1zfEcY6oGh5nQ0NaYEhrZgW2us9bVzGZes/nb/cNoNkAG5l3iAwAJho6gdAvJXq8j4Om4l1XVpuGQIaagBcA95s1kwDv4pGFMadV7J8C2i1lSrPvHtlovDWWuYFjfB/de7zmkKtEuYJIILXACTEEwTdvSRfoucocfRb4X1Xva91NlVwAe3S1p0sa4QYdvmI3N1T+XVatKozh9Y4cH3jWVJa5zZNhadJg56YWs31GNz3Xi+0db3j2TVL26JBdJcw7sdF5mBJubZTvZzlr6lUe7c51JpubM021CQ62QcSfCvK4+m5tSHAs1bPklrTAuSJIjeF6PF+0jns921jGARdgiSAWkuJmQWuKy06/n3MGuoOLAdDLF0zL2uABgXDZBF8kKuA5yGtD6zjqcPC0C+TPzPXAyuX4+sf8XUajXvrQ8tbimzBa4CwJOkwACN7EqvZHjGscXF01ILaTDOlpMEvcfwhXy9p4+nWc65g/h3NqwHUnDS8ASQfi1A7WnzXB+0FVlSrqpuJaQPiMuBMkhxPc9TC7biOD4mhTcxjWupupSRqBaz/YskWBE2vtHbgOYvpHR7prmjT4tRB8W8R+HpN03TnF8q4p9Opqpkj/AGg5buM38l33KONaKTnlraTAYaBAGJtBIuZ69Lr5zwpZq8eqL4AN4MTJxMT2lejS5jUqmmxzrMs2LbYtuYA1bKZsXcfSOErh7GvBs4T/AAl80bNJ/UNJB6J9BsNEiCALC4Hrb8ln4+dBLS2wOqcafxDsVpiD5c0uZIZADWudpBLRIFzOJPkLpxcbwbHPfdc1yfmdQVAC74m30zGjSfCCLHedvDC6CnUDgCMG48vJXyWIUBTjEbpZSkJcEpwTnJbwpVJLVSKFFFOaU0JTQmtC61kwBStOkloBdsDYHzOyp7JtJ/VedwfOGueaTiA8GwBs4Zt0ObZspvRHje0PMhLmnGk2gEtqQW7nBF539Vl4XnrnECLtv4fCCAQQC1sTjFzJsne2DWNcSWfGBDrgSCNQtkkb7R3WLg+JpNOo0H02yC14edQtqA1OEQQNrrnu+1x2vB8W7SXViGAmQDa22/SOmVvBkSuT5G1j6hq1C9xsA18wN3OfYAGSc5my6arxIa0PjwEiTiB/tfbC1nSbjJz/AFe5e5pIAEujJbBB/eN4iQuAdzJxpMovkNpucWubZ4kYicXnr3X0xtRrtIBkVGmI3Fr+S+ee0PBNaQB7sODnAwAywxbUZBvcgXFtlOjl4jHAuGq4kT5Tdd5y7nLavEUqFF5bRbaSIL3C4b2B+t7LhavDubEjLQ4eREhdH7GhorB7iHFsaGBkvc4zZpItF5N+11nNa0z2tNF7zepSqzDqb6djAPiDht06yuXqUiADkH1jsYwbYXUe3XBv/wAl/jDpAIaJJay2RfTcyuaqB9Ilptnu05bI2Nib900xnTKREgOLg3eLmOwJEpatRXpcuFR49xSYX63g2HiOmc9Bv6L6jynjaMCkx7fBLBDpki+lm7oBHr6r5hyzmtSi0spvcNeoECADqAbckT/Q817vIOGps4eu9/vXPsWPpjdpBIBd5Qd7WnapuV4nP6ZpcQ+XAuLnEj49HiMAl0yYgpHJmUjVHvn6WZJhzv8Axhom6rm3F1Kj/wDkJJaNILgGmATEgDPmSZ3WEFRW7mlRgc5tF7nUpkSA2T10jC0+zb2sqe8eNUA6GY1viQJ2AyT2WPjeKDw3TTaxrbAiZMyTJJvck9kzg+Ye7eXhoMi2oB0Otdvl09Cg6f2o55qOh1N1Osaeh4LAGvuL6XOOn4e8dchcbUaNUXzuIXtcLXDtXEVyaji5wcHTF2kghwwdRtGI7283jaweC8/GXX8o3x226oMZXReyFKaksnWAPKLh0wfK0LnV73shxjKVbW9wAgjfJ3AGbD6oO+5kajBrYJ0/E3qOtgZ9F5VGvTbR966oKbqsyXTDiLGG+XSJXocfUe7h3lhDagbqDRDiIGoNItMtz57rg/8A5h1SmKRY0uY3S0nOkEOIIJIm2RGOiVMeZ7wNcRqLhiYi+LX+vdd97O8RUqNL3TpIAA2kWOmwgW87rgHmnAIBDh8QsWm5kjpsI7G6972Z5y5rhTDZBJNhfB8LRgCQPkiu5lR9IhodFnTBtkZHbO/VI4Uv0+PJvEfD0b3jrul8Jx4L3NafEyQfqLjyVqQwpbkxyU5QLJUUhUg0MTQEtqbPT0/tWkYeZPrNIfTAc0DxNJvH/T3z9Fz3FcbTLzUa46S9uum/wODmx4qZzIJEgCV2FCoH2kNMwdVoMdv0XM8+5C94JZp+IugNvvPiLj2sICUeFzSo4Pa7/IFZktIBJuPiAeO0kG85VUuOrFrmtLAwOlsuH/HJIPuy507x6915vFcM6m8sd8TbEdD0VUKZcQNiQMwJvFzjdRXZcnpVHtaymNVIiHucSNRBdLcOAFxmJC6mnWY8+78NmhukWsC7ptcQei8HlBocINJeTUMa76mttOQMZIESk8r9qKTatQwTrfa4ADbEkzAAud0qR5nF0K3Dve2mXPbUaWtiWhhJaQRcxFoNpnyXh82rve8OfU947SJd6YMgXEwuq5N7yq6q4MqVqWpr2NaS0atckOk4Gk+F1pEiMrnef8NTZUcG1Nbg4tMNIk/idqki52QeWDC08DxzqZc5oBdpIa4zLCfxMg2cBN9souW8GyoSHVNJgkDSTqPS0rquUcj4XiOHP4ajBpNUkhhd8QJBi8G4+pRRU+QVf8J1Z5ealVzC4OeIdfwuuJm4GZyuN4kuB0OPwEgATAvcCdl0jqri97KPFEUabA01DqAPhgtbE6QTN7T335ri3S68SLEgzqMnxTvPUohKNhF5F9vNCEVTTJ0zE2nMbTCK9fk/DltJ3EtcWvovBaS2Wu3Il3hJHTvhLp85eaj3Etl7nOLiwOIJEW1TH6SvOY46SNcCx038R2AAET5oWbgmPSb7BBs5rxlSrVc95DiPDqa1obvgMEdb7q6Nal7kt93NUukPJMBoH+oOZtex7b4mTgTfbyuqqZ29MILJzObXn7lC0xcbKCIPVXTZJAGSY6fmg0UqoaCQ46ngtdaYBgkzvN0l7ts3z/CdxZpw0Uw6fxEnJgW0gCIv1meyzIKWvlnEinUa44BExIIG+mCLwsiiDs+Ye1TA402Ug6iz4TPiJIGalzYyZGYGy5Zldge4lktIMNkgScXmbZR8HxH/ANYYHanDs4nAGo2Avuk8XT0vIkGIuCCLDYgkHp6IFlm+0xO3lK1cDx5pCW2cHAi0z1BOzYzGVlNQ6Q3YEmO53PVFQA1Cb+sfXsg+ocC0+6ZqkOcJMmTJvleby57W16tMlxeYdJEWFtrfJM5JXbxHD6HOaHtxpdLmj8Lp69+y8Hi6lSnWfqfS1HwvOoNloyS2bPINolSjriUtyX/kNlrGwZGB+FvUjYI3KUAVSuVSUaWJjVaio8/mvLjUhzXua4CwBgEgyJMfcpHA87pu8DzpqAkECTeYEOi82UUU35RwPFzqJIi5GZxYydylB5iJsc9/NRRaBUmk2mM/MArdybl3vqopyZIdGBcAkXMxiZUUUCKXElgcBubkEiQJ2wQe427oK9ebNJDSB4SZwI9VFFQArZm5O5ufmVp4jjX1Whg8NNjZ0A2kZeRu49VFEHqct5qKfDvoOpB7KgLpMamOA+IH8WAYsvBrhs+EyDfGm+4gK1EClZUUQMfSLQCfxCQga2e1ifkJ2UUQUM9Fu4rgNNCnV1Tqc5pHQgNNvRwUUU0YFYcYiTBUUVFKKKIIoooguVtbzDSxzGtZ42gOcRLpyYJwqUQYiUTR99lFEHps4xraJY0u1agQ4eGwmxHXHoPRY+J4l7ny8y5sN/8AWw88KKKDpeU8V7ukeIqkuc86RGY+LAtkzK95jjAJzF4x6KKLHSqLlFFEH//Z"
          title="Night sky note"
        />
      </Hidden>
    </WithStyles(ForwardRef(Card))>
  </WithStyles(ForwardRef(CardActionArea))>
</WithStyles(ForwardRef(Grid))>
`;
